build-bench-env.sh

```diff
@@ -765,8 +768,12 @@ if test "$setup_redis" = "1"; then
     echo "$devdir/redis-$version_redis already exists; no need to download it"
   else
     wget --no-verbose "http://download.redis.io/releases/redis-$version_redis.tar.gz"
-    tar xzf "redis-$version_redis.tar.gz"
-    rm "./redis-$version_redis.tar.gz"
+    if test "$(sha256sum redis-$version_redis.tar.gz | cut -d ' ' -f 1)" = "$sha256sum_redis"; then
```

Comment:
`sha256sum` is able to perform the check by itself: `echo $sha256sum_redis redis-$version_redis.tar.gz | sha256sum --check --status`

Reply:
Thanks for the hint!